input/profiles/StructureDefinition-publishable-library-cqfm.json

```diff
@@ -140,7 +140,7 @@
         ],
         "mustSupport": true,
         "binding": {
-          "strength": "extensible",
+          "strength": "example",
```

Comment:
No need to change to example binding, because the resolution is to remove the useContext slice qualityProgram.